shell2http.go

```diff
@@ -171,6 +173,7 @@ func getConfig() (cmdHandlers []Command, appConfig Config, err error) {
 	flag.StringVar(&appConfig.shell, "shell", "sh", "custom shell or \"\" for execute without shell")
 	flag.IntVar(&appConfig.cache, "cache", 0, "caching command out (in seconds)")
 	flag.BoolVar(&appConfig.oneThread, "one-thread", false, "run each shell command in one thread")
+	flag.BoolVar(&appConfig.showErrors, "show-errors", false, "returns the standard output even if the command exits with a non-zero exit code")
 	flag.BoolVar(&appConfig.includeStderr, "include-stderr", false, "also returns output written to stderr (default is stdout only)")
```

Comment:
Very minor comment about the doc for these two lines: if imperative style (e.g. `run each shell...`) is preferred, these should probably be `return the standard ...` and `also return ...`, rather than `returns`.